packages/node_modules/@webex/internal-plugin-calendar/src/util.js

```diff
@@ -0,0 +1,14 @@
+const CalendarUtil = {
+
+  // calculate the end time for the meeting based on the duration so it's stored on
+  // the scheduled meeting item, that way client can display start and end without
+  // calculation on their side
+  // gets the start time from server, and the duration, and reformats
+  derive(item) {
```

Comment:
`derive` is a pretty ambiguous function name. please rename to `calculateEndTime` or `formatEndTime`, something like that.

Reply:
you probably also should add a unit test for this file/function. should be pretty straightforward.